dll/win32/shell32/lang/pl-PL.rc

```diff
@@ -37,9 +38,8 @@ BEGIN
         BEGIN
             MENUITEM "", -1, MFT_SEPARATOR
             MENUITEM "&Autorozmieszczanie", FCIDM_SHVIEW_AUTOARRANGE
+            MENUITEM "Align to &Grid", FCIDM_SHVIEW_ALIGNTOGRID
```

Comment:
MENUITEM "Align to &Grid", FCIDM_SHVIEW_ALIGNTOGRID
replace with
MENUITEM "&Wyrównaj do siatki", FCIDM_SHVIEW_ALIGNTOGRID